pkg/errors/v1alpha1/errors.go

```diff
@@ -71,3 +71,20 @@ func WithStack(err error) error {
 		callers(),
 	}
 }
+
+// Cause returns the underlying cause of the error, if present.
```

Comment:
We can have the comments more descriptive to have a better understanding of this,
have picked this from errors package, see if it is better -
```
// Cause returns the underlying cause of the error, if possible.
// An error value has a cause if it implements the following
// interface:
//
//     type causer interface {
//            Cause() error
//     }
//
// If the error does not implement Cause, the original error will
// be returned. If the error is nil, nil will be returned without further
// investigation.
```

Reply:
Done